sql/core/src/test/scala/org/apache/spark/sql/connector/DataSourceV2SQLSuite.scala

```diff
@@ -1735,16 +1735,22 @@ class DataSourceV2SQLSuite
     }
   }
 
-  test("SPARK-33435: REFRESH TABLE should invalidate all caches referencing the table") {
+  test("SPARK-33435/SPARK-33653: REFRESH TABLE should invalidate all caches referencing " +
```

Comment:
Could you add it as a new test case instead of merging into the existing one, please?

Reply:
Sure I was trying to save a few duplicated lines of code :)